packages/amplify-graphql-model-transformer/src/resolvers/rds/resolver.ts

```diff
@@ -195,6 +110,41 @@ export const createRdsLambda = (
   return fn;
 };
 
+export const createLayerVersionCustomResource = (scope: Construct): AwsCustomResource => {
+  const {
+    SQLLayerVersionCustomResourceID,
+    SQLLayerVersionManifestBucket,
+    SQLLayerVersionManifestBucketRegion,
+    SQLLayerVersionManifestKeyPrefix,
+  } = ResourceConstants.RESOURCES;
+
+  const key = Fn.join('', [SQLLayerVersionManifestKeyPrefix, Fn.ref('AWS::Region')]);
```

Comment:
What happens if there is no entry in the manifest for the customer's region? Do we have a check to fail the deployment early-on?

Reply:
It does fail in the deployment, but not terribly early. If there's a good way to resolve the current region at build time I'm open to adding that validation.

Reply:
yeah I don't think there is way to resolve the region in CDK.